.eslintrc.yaml

```diff
@@ -0,0 +1,4 @@
+---
+extends: availity/workflow
+rules:
+  unicorn/prefer-query-selector: off
```

Comment:
lets just add an ignore for this in the eslint lib lol.

Reply:
I removed it for now, and people can add the rule if they prefer until we add to the lint library.